### ISSUE TITLE
Code duplication

### ISSUE THREAD

**Issue by VaretsaK** [roman-k-tech/hearts#3]
Привіт! А навіщо у тебе іде дублювання перевірки на TypeError у обох функціях?

https://github.com/roman-k-tech/hearts/blame/d744e066393d46e234ec291f96eee7881b8e84a7/day2/task_2.py#L10

**Comment by VaretsaK** [comment]
Якщо шо - я не спец) просто в очі кидається. Я можу помилятися) Ось мій гітлаб по марафону https://gitlab.com/k.varetsa/hearts-day-3

**Comment by roman-k-tech** [comment]
@VaretsaK 
Привiт. Це не дублювання, а просто перевiрка в обох фуннкцiя на вiрний тип вхiдного параметра. Чи ти хочеш щоб воно винесено було як окрема функцiя для перевiрки? Це сильно ускладнить код.

**Comment by VaretsaK** [comment]
Ні, я не про окрему функцію. То я шось наплутав) Дивився на відловлення ексепшинів. Начебто зрозумів. Шось складно мені цей марафон дається))
